apps/web/components/getting-started/steps-views/UserProfile.tsx

```diff
@@ -11,16 +13,38 @@ import { trpc } from "@calcom/trpc/react";
 import { Button, Editor, ImageUploader, Label, showToast } from "@calcom/ui";
 import { UserAvatar } from "@calcom/ui";
 
-type FormData = {
-  bio: string;
+import { ZAdminSetupInputSchema } from "/home/upasnasingh/talworx/skillgram/packages/trpc/server/routers/loggedInViewer/setupAdmin.schema";
```

Comment:
Please use relative paths. Absolute paths will not work on the server. Use @calcom/trpc...

Reply:
Sure, Anshul! That was accidentally overlooked, will fix this.

Reply:
Also, please remove the Privacy Policy and Terms notice from the Signup page.